### ISSUE TITLE
Falling Through the map and deleting cform_cache

### ISSUE THREAD

**Issue by traveman** [themrdemonized/xray-monolith#100]
hey so im under the impression this cform_cache thing is for collision with the new 1.5.3 modified exe's

I had a bug i think when i installed a footstep mod or something where i would fall through parts of the map, found the bug in army warehouse i turned the mod off, it stoped happing but then continued in Agra prom and the thing is it continued even when i launch with no mods installed just plane jane 1.5.3 with the modified exe's. saw this new folder and noticed it had folders with the map names deleted it and the bug went away. this has made trouble shooting a nightmare.

so i guess im just putting this here so if anyone is falling through the map in parts may be good to delete this folder. just like the shader cache the file's are recreated on relaunch

if anyone wants i can send over the mod's to recreate it, they are old and i think delisted.

**Comment by themrdemonized** [comment]
Try the latest version, i disabled cform_cache, see if the issue is fixed

**Comment by SVG1** [comment]
Somehow I didn’t notice any acceleration of loading with this cache, but I have an SSD. Maybe you could make enabling cache optional, otherwise it only takes up space?

**Comment by BrokenParity** [comment]
I'm not questioning this bug. But ...
I'm using modexes from 25.08.2024 (it seems to not have this cform_cache, at least i can't find the directory) and it happened to me as well to fall through the floor in some locations.
It has been caused by a mod i have recently found and used. Removing the mod automatically removed the issue.

**Comment by traveman** [comment]
yeah @BrokenParity, looks like you have a version that didn't have the cache so it wouldn't stick for you if the folder isnt showing up in your appdatta. from my knowledge its never been a function b4 1.5.3

weird thing is it fixed it for me while still having the cache but only in the army warehouse, a little while later after removing the problematic mod that caused the issue in the first place. i found the bug again in agroprom on a new save while testing some other stuff, went through whole the process of elimination again up to the point of not even launching through mo2 bug somehow continued with all my mods turned off and went away after clearing the cache. I have a backup of the cache file's b4 i deleted it if it will help at all. if installed should fall through the map right outside of the research institute on the right side of the road.

dont know much about how much work it would be to add but if you decide bring it back and reenable it maybe a button on the launcher that says "reset collision data" on the launcher or something like it is with the shader cache. the only reason i was able to hunt it down was i noticed something new in my appdatta when i went to delete my user.ltx and looked up "cform_cache" in the anomaly discord and saw one person say it has something to do with collision data.

**Comment by SevenChalices** [comment]
> Try the latest version, i disabled cform_cache, see if the issue is fixed

How to re-enable it manually or how to get the directory's content regenerated?.. I believe that without it the game displays only "flat" graphics, i.e. no levels, meshes, lights, etc....

**Comment by themrdemonized** [comment]
Your issue have nothing to do with cform cache. You most probably have issues with conflicting shaders. Reinstall modded exes from scratch and refer to troubleshooting section
